suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -42,9 +47,11 @@ export const discoveryConfigSlice = createSlice({
         toggleAreTestnetsEnabled: state => {
             state.areTestnetsEnabled = !state.areTestnetsEnabled;
         },
-        setDiscoveryStartTimestamp: (state, { payload }: PayloadAction<number | null>) => {
-            state.discoveryStartTimeStamp = payload;
+        setDiscoveryInfo: (state, { payload }: PayloadAction<DiscoveryInfo | null>) => {
+            state.discoveryInfo = payload;
         },
+        // toggling coin one by one
+        // TODO: this might be deleted later in case we always do batch update of enabled coins
```

Comment:
I don't think we want to do that, I would rather remove the TODO keyword, but of course it's just a nitpick

Reply:
Afaik the design expects to update the whole state of enabled coins at once and not one y one (as we have currently). So this should move from toggling single coin to batch update of whole state.

Reply:
I don't think so, we should toggle enabled coins one by one but only start discovery of newly added coins or removal of data related to disabled coins in batch on leaving the coin settings page.

Reply:
🤔 You are right. I'll rewrite it partially to prepare for this

Reply:
8f5b887272